WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBrowserFragment.java

```diff
@@ -386,16 +386,21 @@ public void setRefreshing(boolean refreshing) {
         }
     }
 
-    private void setEmptyViewVisible(boolean visible) {
+    private void updateDisplay() {
         if (!isAdded() || getView() == null) {
             return;
         }
-        mEmptyView.setVisibility(visible ? RelativeLayout.VISIBLE : RelativeLayout.GONE);
-        mActionableEmptyView.setVisibility(visible ? View.GONE : View.VISIBLE);
-        mGridView.setVisibility(visible ? View.GONE : View.VISIBLE);
-        if (visible && !NetworkUtils.isNetworkAvailable(getActivity())) {
-            mEmptyTextView.setText(R.string.no_network_title);
+
+        boolean hasThemes = getAdapter().getUnfilteredCount() > 0;
+        boolean hasVisibleThemes = getAdapter().getCount() > 0;
+        boolean hasNoMatchingThemes = hasThemes && !hasVisibleThemes;
+
+        mNoThemesView.setVisibility(!hasThemes ? RelativeLayout.VISIBLE : RelativeLayout.GONE);
```

Comment:
I know that the original code used `RelativeLayout.VISIBLE/GONE`, but do you mind switching them to `View.GONE/VISIBLE` for consistency with the rest of codebase?

Reply:
Updated to import the `View` constants instead of the `RelativeLayout` constants.